src/kOS/Module/kOSProcessor.cs

```diff
@@ -528,8 +528,15 @@ public void SetMode(ProcessorModes newProcessorMode)
                 switch (newProcessorMode)
                 {
                     case ProcessorModes.READY:
-                        if ((ProcessorMode == ProcessorModes.STARVED || ProcessorMode == ProcessorModes.OFF) && shared.Cpu != null)
-                            shared.Cpu.Boot();
+                        if (Config.Instance.StartOnArchive)
```

Comment:
We actually already check to make sure it has changed with https://github.com/hvacengi/KOS/blob/update-setmode/src/kOS/Module/kOSProcessor.cs#L526 if we get a READY call followed by another READY call it will just be skipped.